i18n/en.json

```
@@ -4,22 +4,22 @@
 	"ABOUT": "ABOUT",
 	"DOCS": "DOCS",
 	"HYDRATION": "HYDRATION",
-	"contribute": "Contribute",
-	"supportingTheLongTail": "Supporting The Long Tail Of Cryptoassets And Experimenting With Financial Applications And Governance.",
+	"contribute": "Crowdloan",
+	"supportingTheLongTail": "Flexible Liquidity for Young Cryptoassets",
 	"aboutTheTeam": "About the team",
 	"launchApp": "launchApp",
-	"infobox-title-1": "Launchpad for Tokens: Liquidity Bootstrapping Pools",
-	"infobox-title-2": "Cross-Chain Capabilities",
-	"infobox-title-3": "Supporting Long Tail of Assets",
-	"infobox-content-1": "Traditional xyk models are not ideal for a price discovery of new assets. We are building a product that makes bootstrapping liquidity easier and efficient.",
-	"infobox-content-2": "Kusama enables cross-chain communication and Basilisk transforms that into cross chain swaps, composable & compounding liquidity mining incentives",
-	"infobox-content-3": "Build to be flexible and facilitate liquidity for many tokens. Each token is different. Varying maturity of tokens requires different curves to optimize its liquidity.",
+	"infobox-title-1": "Fair Token Distribution Platform",
+	"infobox-title-2": "Modular AMM Design",
+	"infobox-title-3": "Community first",
```

Should this be an upper case F for consistency? The render appears to be all caps so I would expect this is semantics.

```suggestion
	"infobox-title-3": "Community First",
```

---

yes